src/app/run/pipelines/pipelines.component.html

```diff
@@ -1,3 +1,7 @@
 <div id="pipelines" class="container content">
-  <h1>TODO</h1>
-</div>
\ No newline at end of file
+    <h1>TODO</h1>
+    <div class="embed-responsive embed-responsive-16by9">
+        <iframe class="embed-responsive-item" width="100%" height="800" src="http://localhost:3001/"
```

Comment:
Why are we using an iframe? This should be a component, even if it is an external one.